sdk/storage/azblob/appendblob/client_test.go

```diff
@@ -141,7 +141,7 @@ func (s *AppendBlobUnrecordedTestsSuite) TestAppendBlockWithAutoGeneratedCRC64()
 	// _require.Equal(resp.RawResponse.StatusCode, 201)
 
 	// test append block with valid CRC64 value
-	readerToBody, body := testcommon.GetRandomDataAndReader(1024)
+	readerToBody, body := testcommon.GetDataAndReader(testName, 1024)
```

Comment:
I noticed some tests were left unrecorded and some are now recorded. Is there a reason why some of these tests (like this one, ignoring the tests for APIs using SAS) is left unrecorded after updating to use the seed random data?

Reply:
CRC64 had some problem with specific configurations of OS + Go version. Some are SAS in copy source header related.